.env.example

```diff
@@ -1,8 +1,9 @@
 NODE_ENV="development"
 
-DATABASE_URL="postgresql://developer:dev_pass@localhost:5432/mayak?schema=public"
+DATABASE_URL=""
+DIRECT_DATABASE_URL=""
 
-ADMIN_USERNAME="admin"
-ADMIN_PASSWORD="admin"
+ADMIN_USERNAME=""
+ADMIN_PASSWORD=""
```

Comment:
If it's empty, would fail to start an app or would it allow empty login and password? )

Reply:
it will fail to start

Reply:
actually its probably fail on admin route, rather than just fail everything